### DISCUSSION TITLE
Logs of steps are only printed once step is completed or canceled - TMATE no longer working

### DISCUSSION THREAD

**Discussion by CloseChoice** [community/community#66515] [Actions]
### Select Topic Area

Bug

### Body

Hi, 
I was previously debugging my github actions pipelines with [tmate](https://github.com/mxschmitt/action-tmate), which provides an ssh command to get into the runner. It seems like there was a change made that the logs of the currently running step are not printed to the console, they just appear once the step is finished:

![image](https://github.com/community/community/assets/31857876/f1410f60-bd4f-44f8-94a9-73f17fd78614)
I cannot click the dropdown arrow on the currently running job `Setup tmate session`. 

Previously I was able to access the logs of tmate while the step was running which enabled me to get the correct ssh command to log into the server. Without this feature I cannot use tmate anymore which is a massive downgrade of github actions to me.

Is this done intentionally?

**Comment by benbrackenc74** [comment]
I can confirm this behavior with all of our Actions. I have submitted a ticket to GH for it.

**Comment by corang** [comment]
I'm also still seeing this behaviour @benbrackenc74 is there a place I can +1 that ticket? Seeing this on all actions in firefox and edge, coworker has it working with chrome though.

**Comment by benbrackenc74** [comment]
I just have a personal ticket open with GH, don't think there is an official issue yet. I also saw this related issue:
https://github.com/mxschmitt/action-tmate/issues/174

**Comment by corang** [comment]
Oh I guess I should clarify, this isn't with tmate, this is with any job regardless of what it is.

**Comment by benbrackenc74** [comment]
Yes, same here, just wanted to cross-link.

**Comment by benbrackenc74** [comment]
This was fixed for me with an update to Firefox.
